QUEST_LV_0100.tsv

```diff
@@ -585,58 +585,58 @@ QUEST_LV_0100_20150317_000584	$Miners' Village Mayor
-QUEST_LV_0100_20150317_000588	Mausoleum Foundation Stone
-QUEST_LV_0100_20150317_000589	I pass my words to the revelator who comes here.{nl}I, Zachariel, sleep here to protect the goal of the Goddess. 
-QUEST_LV_0100_20150317_000590	If something vicious comes here, protect here from it by waking the sleeping protector.{nl}{nl}
-QUEST_LV_0100_20150317_000591	The manual to use Mausoleum Cube
-QUEST_LV_0100_20150317_000592	The protector who is polluted with the vicious atmosphere will forget his responsibility to protect the Mausoleum.{nl}We can defeat the protector near the cube.
-QUEST_LV_0100_20150317_000593	The manual of the protection system
-QUEST_LV_0100_20150317_000594	The protecting device that forgot its mission to protect the Mausoleum is deserved to be destroyed.
-QUEST_LV_0100_20150317_000595	The role of the protector
-QUEST_LV_0100_20150317_000596	If we dedicate the power supply of the protector that forgot its mission, we can return it to the part of the Mausoleum.
-QUEST_LV_0100_20150317_000597	The statue of the protector
-QUEST_LV_0100_20150317_000598	The protectors are polluted with vicious energy.{nl}Destroy the polluted protectors to stop spreading.
-QUEST_LV_0100_20150317_000599	The vicious beings are trying to take on from the roots of the tomb.{nl}If all controlling magics get polluted, we can't protect the tomb anymore.
-QUEST_LV_0100_20150317_000600	The protectors are being polluted like a plague.{nl}We need to look for a special way.
-QUEST_LV_0100_20150317_000601	We will destroy it when the polluted protectors are in sleep.{nl}Bring the condensed magic from the jar.
+QUEST_LV_0100_20150317_000588	$Mausoleum Foundation Stone
+QUEST_LV_0100_20150317_000589	$I pass my words to the Revelator who comes here.{nl}I, Zachariel, sleep here to protect the goal of the Goddess. 
+QUEST_LV_0100_20150317_000590	$If evil forces attack this place, please protect this place by waking the sleeping protectors.{nl}{nl}
+QUEST_LV_0100_20150317_000591	$Mausoleum Cube Manual
+QUEST_LV_0100_20150317_000592	$A protector who is corrupted by evil will forget its responsibility to protect the Mausoleum.{nl}Defeat these protectors near the large cubes.
+QUEST_LV_0100_20150317_000593	$Defense System Manual
+QUEST_LV_0100_20150317_000594	$A protector that forgot its mission to protect the Mausoleum deserves to be destroyed.
+QUEST_LV_0100_20150317_000595	$Information on Protectors
+QUEST_LV_0100_20150317_000596	$By recycling the power supply of protectors that forgot their mission, we can return them to be part of the Mausoleum.
+QUEST_LV_0100_20150317_000597	$Protector Statue
+QUEST_LV_0100_20150317_000598	$The protectors are corrupted with evil energy.{nl}Destroy the corrupted protectors to stop its spreading.
+QUEST_LV_0100_20150317_000599	$The demons are trying to take over the roots of the tomb.{nl}If all controlling magics get corrupted, we can't protect the tomb anymore.
```

Comment:
of the tomb -> of the Royal Mausoleum.
magics -> magic
the tomb anymore -> the Royal Mausoleum anymore